apis/go.mod

```diff
@@ -4,17 +4,16 @@ go 1.24.0
 
 require (
 	github.com/stretchr/testify v1.10.0
-	k8s.io/api v0.32.6
-	k8s.io/apimachinery v0.32.6
-	sigs.k8s.io/controller-runtime v0.20.4
+	k8s.io/api v0.33.0
```

Comment:
You're bumping these to v0.33.2 in another PR, maybe combine them here?

Reply:
okay